packages/@react-spectrum/view/stories/View.stories.tsx

```diff
@@ -9,33 +9,46 @@
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
-
 import {Button} from '@react-spectrum/button';
+import {ComponentMeta, ComponentStoryObj} from '@storybook/react';
 import React from 'react';
-import {storiesOf} from '@storybook/react';
 import {View} from '../';
 
-storiesOf('View', module)
-  .add(
-    'name me',
-    () => (
-      <div style={{display: 'flex'}}>
-        <View 
-          backgroundColor="negative"
-          width="single-line-width"
-          height="size-500"
-          elementType="span" />
-        <View 
-          backgroundColor="positive"
-          width="size-500"
-          height="size-500"
-          marginStart="size-250"
-          borderColor="default"
-          borderWidth="thin" />
-        <Button variant="primary" marginStart="size-250">Test</Button>
-      </div>
-    )
+type ViewStory = ComponentStoryObj<typeof View>;
+
+let meta = {
+  title: 'View',
+  component: View
+} as ComponentMeta<typeof View>;
+
+export default meta;
+
+export const NameMe: ViewStory = {
+  render: () => (
```

Comment:
I think we can probably completely get rid of all stories for view and only have chromatic ones, if we want to keep any View ones, we can add some controls

Reply:
How I would do your suggestion is to move both stories over to chromatic and not have any storybook stories. These two stories seemed to test passing in `cacl()` and special strings for width.

I looked at all the props and there were a ton that were mostly formatting related, I didn't want to do argTypes on that many. It seemed like a lot of work for not much return. I could see creating a single story with a number of these props and have Storybook autogenerate controls.